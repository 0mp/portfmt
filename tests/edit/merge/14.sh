cat <<EOD | ${PORTEDIT} merge 14.in | diff -L 14.expected -L 14.actual -u 14.expected -
DISTVERSIONSUFFIX=	-gababab
DISTVERSION=	1
DISTVERSIONPREFIX=	v
EOD
