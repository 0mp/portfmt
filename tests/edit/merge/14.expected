DISTVERSIONPREFIX=	v
DISTVERSION=	1
DISTVERSIONSUFFIX=	-gababab
